package.json

```diff
@@ -20,6 +20,12 @@
   },
   "homepage": "https://github.com/Code-For-Chicago/Voma-frontend#readme",
   "dependencies": {
```

Comment:
Can you uninstall these packages here? They belong in the `client/package.json`.